hooks/use-chat.ts

```
@@ -640,14 +872,127 @@ export function useChat() {
+    }
+  };
+
+  const loadProjectThreads = async (projectId: string) => {
+    try {
+      setIsLoading(true);
+      setError(null);
+
+      // Load project data which includes threads
+      const projectData = await loadProjectData(projectId);
+
+      if (projectData && projectData.threads) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)